apigee-cf-service-broker/api/service_instances.js

```diff
@@ -120,6 +120,38 @@ function actionValidate(req, res, next) {
   }
 }
 
+function deriveProtocol(params){
+  const proto = params.protocol ? params.protocol.toString().trim().toLowerCase() : ''
+  const ret = {
+    any: false,
+    errors: []
+  }
+  if (proto === "http" || proto === "https"){
+    ret["protocol"] = proto
+    ret.any = true
+  }
+  else if (proto){
+    ret.errors.push(proto)
```

Comment:
This adds the invalid parameter after it has been lowercased, not as it was typed.